core/tests/test_utils.py

```diff
@@ -622,6 +622,9 @@ def process_and_flush_pending_tasks(self, queue_name=None):
                 if task.url == '/_ah/queue/deferred':
                     from google.appengine.ext import deferred
                     deferred.run(task.payload)
+                elif feconf.EMAILS_TASK_PREFIX in task.url:
+                    # Flush task if task is of emails taskqueue.
```

Comment:
Sounds good, thanks @prasanna08! Please let me know when you want another review.